### PR TITLE
Working with Bitmarks: Full Edit

### PR DESCRIPTION
**Pull request by shannona**
Edit of all sections from Working with Bitmarks, including better incorporation of payment-for-bitmark-cli

### CODE REVIEW

File: learning-bitmark/quick-start/working-with-bitmarks/README.md [hunk 1]
```diff
@@ -1,11 +1,23 @@
-# Working with bitmarks
+# Working with Bitmarks
 
-## [Creating Bitmark Account](creating-bitmark-account.md#creating-bitmark-account)
+Interacting with Bitmark requires the creation of a Bitmark Account. Afterward, users can register Bitmark Certificates for their digital properties, which they may later transfer to other users or divide into Bitmark shares.
+
+Most work with the Bitmark Property System can be done via one of three methods: the [Bitmark App](https://a.bitmark.com/) (for iOS or Android), the [Bitmark SDK](https://github.com/bitmark-inc) (for a variety of programming languages), or the Bitmark CLI (available as part of the [Bitcoin node](https://github.com/bitmark-inc/docs/blob/master/learning-bitmark/quick-start/simple-solution-for-node-setup.md). 
+
```
Comment (vienlt) [inline learning-bitmark/quick-start/working-with-bitmarks/README.md]: > or the Bitmark CLI (available as part of the [Bitcoin node]
Do you mean `Bitmark node`? And it misses a ')'

File: learning-bitmark/quick-start/working-with-bitmarks/README.md [hunk 2]
```diff
@@ -14,35 +26,46 @@
...
   
 * [Prerequisites](transferring-bitmarks.md#prerequisites)
-* [Transferring bitmarks using Bitmark App](transferring-bitmarks.md#transferring-bitmarks-using-bitmark-app)
-* [Transferring bitmarks using Bitmark SDK](transferring-bitmarks.md#transferring-bitmarks-using-bitmark-sdk)
-* [Transferring bitmarks using Bitmark CLI](transferring-bitmarks.md#transferring-bitmarks-using-bitmark-cli)
-* [Explore the Bitmark transactions using the Bitmark Registry website](transferring-bitmarks.md#explore-the-bitmark-transactions-using-the-bitmark-registry-website)
+* [Transferring Bitmarks using the Bitmark App](transferring-bitmarks.md#transferring-bitmarks-using-the-bitmark-app)
+* [Transferring Bitmarks using the Bitmark SDK](transferring-bitmarks.md#transferring-bitmarks-using-the-bitmark-sdk)
+* [Transferring Bitmarks using the Bitmark CLI](transferring-bitmarks.md#transferring-bitmarks-using-the-bitmark-cli)
+* [Exploring Bitmark transactions using the Bitmark Registry website](transferring-bitmarks.md#exploring-bitmark-transactions-using-the-bitmark-registry-website)
```
Comment (vienlt) [inline learning-bitmark/quick-start/working-with-bitmarks/README.md]: In the Bitmark convention, `Bitmark` is using to refer the company name or the Bitmark System. And `bitmark` is using to refer to a record on the blockchain which is a chain of issue record to the last transfer record.

**Review by vienlt — APPROVED** [review]
The editing looks good to me. Only 2 minor comments have been added.

**Comment by vienlt** [comment]
@shannona: There are commits on master to support for the deployment so they leads to the conflicts with your branch. Could you resolve the conflicts and merge your branch to master? Thanks.

**Comment by araujobsd** [comment]
@vienlt @jollyjoker992 I think the conflict happened after @jollyjoker992 moved the files to pages directory!!! Is it possible for you both to double check this PR and merge it?

File: learning-bitmark/quick-start/working-with-bitmarks/README.md (hunk 1 above)
Comment (shannona) [reply learning-bitmark/quick-start/working-with-bitmarks/README.md]: Yep, thanks.

File: learning-bitmark/quick-start/working-with-bitmarks/README.md (hunk 2 above)
Comment (shannona) [reply learning-bitmark/quick-start/working-with-bitmarks/README.md]: Thanks, I fixed it in the table of contents, and found a few other instances in the text.

**Comment by shannona** [comment]
@vienlt @araujobsd I've resolved the comments and was going to work through the conflicts, but it looks like I've lost write access some time in the last week or so:

This branch has conflicts that must be resolved
Only those with write access to this repository can merge pull requests.
You’re not authorized to merge this pull request.

**Comment by vienlt** [comment]
@araujobsd: Could you check Shannon's permissions on this repo? Thanks.
@shannona: I merge the master branch to this branch and resolve the conflicts.

**Pull request merged**